data/projects.json

```diff
@@ -138,6 +138,16 @@
         "level": "basic",
         "site": "",
         "tags": "car-game,c,c++,graphics-project"
+    },
+    {
+        "name": "Tamagotchi-Game",
```

Comment:
The repo name doesn't exist, please check the repo name or its visibility
![image](https://user-images.githubusercontent.com/37890026/124389989-a462db00-dcb7-11eb-9a65-7014c067f958.png)

Reply:
I haven't finished the game yet and it's private. I removed the project and will come back and add it when it's finished. Thank you!